netket/jax/sharding.py

```diff
@@ -314,6 +315,7 @@ def sharding_decorator(f, sharded_args_tree, reduction_op_tree=False):
             containing True/False indicating that each input in the argumens of f is:
                 True: sharded on axis 0 (True)
                 False: assumed to be replicated
+                'key': A jax.random.PRNGKey. Assumes jax_threefry_partitionable is enabled.
```

Comment:
does this also work with `jax.random.key()`? because `PRNGKey` will eventually disappear (and nowadays I use `key`in my code)